publish-to-maven-central/properties.sh

```diff
@@ -23,9 +22,8 @@ FILE_ECLIPSE=${DROPS4}/${SDK_BUILD_DIR}/eclipse-SDK-${SDK_VERSION}-linux-gtk-x86
 ECJ_VERSION=3.30.0
 
 # AGGREGATOR:
-IU_AGG_PRODUCT=org.eclipse.cbi.p2repo.cli.product
 # URL_AGG_UPDATES=https://download.eclipse.org/cbi/updates/aggregator/headless/4.13/I20200825-1209/
-URL_AGG_UPDATES=https://download.eclipse.org/cbi/updates/p2-aggregator/tools/nightly/latest/
+URL_AGG_UPDATES=https://download.eclipse.org/cbi/updates/p2-aggregator/products/nightly/
```

Comment:
This will work, but it is a composite with 5 children.   

This will be a composite with a single child:

https://download.eclipse.org/cbi/updates/p2-aggregator/products/nightly/latest

So it is a little cheaper to load...

Reply:
OK, changed it. Thanks.